packages/build/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I think we should be using top-level `.prettierrc` from the monorepo, not per-package configs. Please remove this file.

Reply:
This is intentional as I want to make sure the scripts are formatted with `es5`.

Reply:
I see, makes sense 👍